catalogue_of_life/README.md

```diff
@@ -0,0 +1,207 @@
+# Catalogue of Life example
+
+[Catalogue of Life](https://www.catalogueoflife.org/) is a database of over 4.5 million currently known taxa in biology,
+compiled from over a hundred different sources. 
+
+In this example we use [TypeDB-Loader](https://github.com/typedb-osi/typedb-loader) to load this dataset into TypeDB
+and leverage its inference capabilities.
```

Comment:
This is a tiny bit unclear - `loader` doesn't have any inference capabilities!
I'd probably reword it as `into TypeDB. Queries against TypeDB can then leverage rule-based inference capabilities.` or something a bit less awkward than that :P